tests/Integration/Behaviour/Features/Scenario/Order/order_odd_tax.feature

```diff
@@ -1,5 +1,6 @@
 # ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s order --tags order-odd-tax
 @reset-database-before-feature
+@reset-context-before-feature
```

Comment:
Another detail you set the tag here `@reset-context-before-feature` I guess because the context was messed up by the previous feature `order_multi_shop.feature` (I assume so because this is where you create shops and all)

Then I think it's the `order_multi_shop.feature` which should have a tag and clean after it's done This way it avoids breaking other features, and if one day we introduce a new feature that is executed between `order_multi_shop.feature` and `order_odd_tax.feature` It won't need to be added this cleaning tag

Reply:
Fixed.